docker-compose.yaml

```diff
@@ -177,7 +177,7 @@ services:
       - OPENREPLAY=${OPENREPLAY:-}
       - PAPERCUPS_STORYTIME=${PAPERCUPS_STORYTIME:-}
   airbyte-temporal:
-    image: airbyte/temporal:${VERSION}
+    image: temporalio/auto-setup:1.7.0 # todo: revert
```

Comment:
I need to revert this before merging, this is only so it's easier to build locally on M1.